packages/teleport/src/services/databases/types.ts

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 export interface Database {
   name: string;
   desc: string;
-  type: string;
+  displayText: string;
+  protocol: 'postgres' | 'mysql';
```

Comment:
we already typed the protocol, so lets re-use that

Reply:
Good catch, it's better to type as you suggested rather than redefining the same type more than once. This way in case we for example add `mongodb` as a protocol in the future, we'll only have to change the `DbProtocol` type definition once and it will apply everywhere it's used